File: test/fixtures/parity/parity.rb
def classify(items, limit)
  total = 0
  for item in items
    if item > limit && item % 2 == 0
      total += item
    elsif item < 0
      total -= 1
    else
      total += 1
    end
  end
  case total % 3
  when 0
    "zero"
  when 1
    "one"
  else
    "other"
  end
end

File: test/fixtures/parity/parity.kt
fun classify(items: IntArray, limit: Int): String {
    var total = 0
    for (item in items) {
        if (item > limit && item % 2 == 0) {
            total += item
        } else if (item < 0) {
            total -= 1
        } else {
            total += 1
        }
    }
    when (total % 3) {
        0 -> return "zero"
        1 -> return "one"
        else -> return "other"
    }
}
